houston/ardu/sandbox.py

```diff
@@ -213,86 +221,189 @@ def run(self, commands: Sequence[Command]) -> 'MissionOutcome':
             logger.debug("Mission uploaded")
             vcmds.wait_ready()
 
-            mylock = threading.Lock()
-            event = threading.Event()
+            wp_lock = threading.Lock()
+            wp_event = threading.Event()
 
-            wp_state = {}
-            last_wp = [-1]
+            wp_state = {}  # Dict[int, Tuple[State, float]]
```

Comment:
`wp_to_state` would be a better variable name. A short comment above would also be great.